uint16 tag_id
uint16 anchor_id
uint32 dist_mm
float32 power
int16 magnetometer_x
int16 magnetometer_y
int16 magnetometer_z
